extensions/amp-story/1.0/page-advancement.js

```diff
@@ -291,19 +293,13 @@ class ManualAdvancement extends AdvancementConfig {
     };
   }
 
-  /** @override */
```

Comment:
To keep support for the paused state, could we keep these `start` and `stop` methods, and call them from the `story.onPausedStateUpdate_` method to block the navigation?